### PR TITLE
add adtk

### PR DESCRIPTION
**Pull request by rpanai**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/adtk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+    - pandas >=0.23
+    - numpy >=1.15
+    - matplotlib >=3.0
+    - scikit-learn >=0.20
+    - statsmodels >=0.9
```
Comment (marcelotrevisani) [inline recipes/adtk/meta.yaml]:
```suggestion
    - statsmodels <0.11,>=0.9
```

File: recipes/adtk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - packaging >=17.0
+    - tabulate >=0.8
+
+test:
+  imports:
+    - adtk
+
+about:
+  home: https://github.com/arundo/adtk
+  license: "Mozilla Public 2.0 (MPL 2.0)"
```
Comment (marcelotrevisani) [inline recipes/adtk/meta.yaml]:
```suggestion
  license: "MPL-2.0"
```

File: recipes/adtk/meta.yaml (hunk 1 above)
Comment (rpanai) [reply recipes/adtk/meta.yaml]: Why this?

Comment (marcelotrevisani) [reply recipes/adtk/meta.yaml]: it is in their pypi metadata this restriction, but if you think it is not necessary that is fine then

File: recipes/adtk/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+    - pandas >=0.23
+    - numpy >=1.15
+    - matplotlib >=3.0
```
Comment (scopatz) [inline recipes/adtk/meta.yaml]: This should be matplotlib-base

Comment (rpanai) [reply recipes/adtk/meta.yaml]: @scopatz done!

**Comment by rpanai** [comment]
There is anything that prevent this from being merged?

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**